coalib/collecting/Collectors.py

```diff
@@ -170,7 +170,9 @@ def collect_bears(bear_dirs, bear_globs, kinds, log_printer,
 
     if warn_if_unused_glob:
         _warn_if_unused_glob(log_printer, bear_globs, bear_globs_with_bears,
-                             "No bears were found matching '{}'.")
+                             "No bears were found matching '{}'. Make sure you "
+                             "typed the name correctly or install it from here "
+                             "https://github.com/coala-analyzer/coala-bears.")
```

Comment:
Shouldn't the user be told about the `coala-install` tool to install bears using that ? CC @Adrianzatreanu 
The install bears from git/using coala-bears is not a very good approach as it installs a bucket load of bears.

Also, they may have installed only PEP8Bear, but not PyLintBear using he new approach of installing bears. So, it's not necessary that all bears are installed.